src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -507,5 +507,18 @@ public static int countEmptyStrings(String... strings) {
      */
     public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
-    }  
-}
\ No newline at end of file
+    } 
+    
+    /**
+     * Returns a String array after removing white spaces preceding or
+     * succeeding any string in that array
```

Comment:
`that array` seems wrong to me. Maybe we can mention the parameter `stringsToTrim` or just `input array`.